### PR TITLE
Show `x-error` response header when config change fails

### PR DESCRIPTION
**Pull request by dominique-pfister**
Now looks as follows:

Screen Shot 2022-06-01 at 08 18 11

### CODE REVIEW

File: src/sidekick/module.js [hunk 1]
```diff
@@ -892,12 +892,21 @@
...
-            console.error(resp);
-            sk.showModal({
-              css: 'modal-preview-failure',
-              sticky: true,
-              level: 0,
-            });
+            if (status.webPath.startsWith('/.helix/') && resp.error) {
+              // show detail message only in power-user mode
+              sk.showModal({
+                message: `Previewing failed. ${resp.error}`,
```
Comment (dominique-pfister) [inline src/sidekick/module.js:898]: `resp.error` will be in english, anyway

[the remark this answers is not in the document]
Comment (rofe) [reply src/sidekick/module.js:898]: The prefix could even be localized in CSS, for consistency...

File: test/preview.test.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ describe('Test preview plugin', () => {
     assert.ok(notification.className.includes('modal-config-success'), `Unexpected notification classes: ${notification.className}`);
   }).timeout(IT_DEFAULT_TIMEOUT);
 
-  it('Edit-specific preview plugin handles /.helix/* error message from server', async () => {
+  it.only('Edit-specific preview plugin shows /.helix/* error message from server', async () => {
```
Comment (dominique-pfister) [inline test/preview.test.js:133]: Just for testing, I guess - did you think to remove it afterwards?

Comment (rofe) [reply test/preview.test.js]: Good catch 😉

**Pull request merged**